### ISSUE TITLE
Stage 2 training not converging 

### ISSUE THREAD

**Issue by yellowwoods12** [yongzx/PyTorch-PaletteNet#9]
Hi, I tried training the stage 2 of the model using gan loss and euclidean loss as mentioned in your jupyter notebook. But the losses doesn't seem to converge, gan loss kept on increasing or fluctuating in the same range. My dataset is quite large(over 10k images) so it shouldn't be a dataset issue. Even when I tried overfitting the model, it didn't overfit. Attaching an overfitting experiment screenshot. This is for 10 images trained for 1000 epochs for the stage 2 part. 

<img width="638" alt="Screenshot 2023-07-05 at 3 08 29 PM" src="https://github.com/yongzx/PaletteNet-PyTorch/assets/31931348/b6cd5ec8-320a-4705-a98b-4370f3b85eb0">

@yongzx Do you have any suggestions on why this might be happening or what can I try to fix it?

**Comment by Y1pLau** [comment]
I also has this problem, @yongzx  have any solution? Thank you!
